fog/ledger/server/src/bin/router.rs

```diff
@@ -16,6 +16,28 @@ fn main() {
     mc_common::setup_panic_handler();
     let config = LedgerRouterConfig::parse();
 
+    let _runtime = if mc_util_telemetry::telemetry_enabled() {
```

Comment:
As the file name says this is the router, it didn't support tracing before, but now it should